manage.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```

Comment:
One tip is to use django-admin startproject to generate the initially required files of Django, you would require the current version installed, the 1.11.